users/views.py

```diff
@@ -1,3 +1,65 @@
-from django.shortcuts import render
+import json, re, bcrypt, jwt
 
-# Create your views here.
+from django.views        import View
+from django.http         import JsonResponse, request
+
+from users.models        import User, UserColor
+from products.models     import Size, Color
+
+from gaebalance.settings import SECRET_KEY  
+
+email_ragular    = re.compile('^[a-zA-Z0-9+-_.]+@[a-zA-Z0-9-]+\.[a-zA-Z0-9-.]+$')
+password_regular = re.compile('^(?=.*[A-Za-z])(?=.*\d)(?=.*[$@$!%*#?&])[A-Za-z\d$@$!%*#?&]{8,12}$')
+
+class JoinView(View):
+    def post(self,request):
+        try:
+            data =json.loads(request.body)
+            bcrypt_password = bcrypt.hashpw(data["password"].encode("utf-8"),bcrypt.gensalt()). decode("utf-8")
+           
+            if (User.objects.filter(email=data["email"]).exists()) or (User.objects.filter(phone_number=data['phone_number']).exists()):
+                return JsonResponse ({"MESSAGE": "EXIST_DATA"}, status = 400)
+
+            if (not email_ragular.match(data["email"])) or (not password_regular.match(data["password"])):
+                return JsonResponse ({"MESSAGE":"INVALID_FORMAT"}, status = 400)	
+  
+       	    user = User.objects.create(
+				name         = data["name"],
+				gender       = data["gender"],
+				birth_date   = data["birth_date"],
+				phone_number = data["phone_number"],
+				email        = data["email"],
+				password     = bcrypt_password,
+				address      = data["address"],
+				size         = Size.objects.get(id=data["size_id"])
+			)  
+		    
+            UserColor.objects.bulk_create([ UserColor(user=User.objects.get(id=user.id), color=Color.objects.get(id=color_id)) for color_id in data["colors_id"]])
```

Comment:
bulk_create 성공 하셨군요!! 잘 하셨어요~!!
1가지만 더 알려드리면 우리는 이미 user을 생성하고 user라는 변수에 생성된 유저를 담았잖아요. 따라서 아래와 같이도 사용할 수 있습니다.
```suggestion
            UserColor.objects.bulk_create([ UserColor(user=user, color=Color.objects.get(id=color_id)) for color_id in data["colors_id"]])
```